app/src/debug/res/xml/shortcuts.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shortcuts xmlns:android="http://schemas.android.com/apk/res/android">
+  <shortcut
+      android:enabled="true"
+      android:icon="@drawable/ic_shortcut_editor"
+      android:shortcutDisabledMessage="@string/label_shortcut_editor_disabled"
+      android:shortcutId="@string/shortcut_id_open_editor"
+      android:shortcutLongLabel="@string/label_shortcut_editor_longlabel"
+      android:shortcutShortLabel="@string/label_shortcut_editor_shortlabel">
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: Attribute shortcutShortLabel is only used in API level 25 and higher (current min is 21) [UnusedAttribute]